app/controllers/admin/requests_controller.rb

```diff
@@ -8,6 +8,8 @@ def index
             .with_last_version
             .order(updated_at: :DESC)
 
+          @requests = @requests.tagged_with(params[:directory], on: :directories) if params[:directory].present?
```

Comment:
I was really confused for a minute because looking at this I thought  _won't it affect the counts for all services below?_ and then I went to see what it looks like on the page and it all looks fine... so I checked and I don't think we're using any of those counts anywhere!

Could you pls confirm and if so, I think we can delete lines 13-33 🥳